core/src/apps/bitcoin/get_public_key.py

```diff
@@ -110,3 +116,23 @@ async def get_public_key(
         xpub=node_xpub,
         root_fingerprint=keychain.root_fingerprint(),
     )
+
+
+def _xpub_descriptor(
+    node_xpub: str,
+    address_n: list[int],
+    script_type: InputScriptType,
+    fingerprint: int,
+):
```

Comment:
missing type hint `-> str` but I wonder why style check doesn't complain :thinking:

Reply:
That is weird, good catch, fixed in [5b8865a](https://github.com/trezor/trezor-firmware/pull/3505/commits/5b8865a4a94f5bfc21c01ed6b2b6feda5ff41ea0).